python-avd/pyavd/_eos_designs/schema/schema_fragments/management_eapi.schema.yml

```
@@ -25,3 +25,44 @@ keys:
         default: true
       default_services:
         type: bool
+      vrfs:
+        # Todo: For 6.0 default should use use_default_mgmt_method_vrf
+        default:
+          - name: use_mgmt_interface_vrf
+            enabled: true
+        type: list
+        primary_key: name
+        description: |-
+          Note: For backward compatibility, the default behavior does not enforce `mgmt_ip` to be present in VRFs.
+          If this enforcement is desired, users must explicitly define VRFs. This behavior applies up to AVD version 6.0.
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: |-
+                VRF name.
+                The value will be interpreted according to these rules:
+                - `use_mgmt_interface_vrf` will configure the Management EAPI ACL under the VRF set with `mgmt_interface_vrf`.
+                  An error will be raised if `mgmt_ip` or `ipv6_mgmt_ip` are not configured for the device.
+                - `use_inband_mgmt_vrf` will configure the Management EAPI ACL under the VRF set with `inband_mgmt_vrf`.
+                  An error will be raised if inband management is not configured for the device.
+                - `use_default_mgmt_method_vrf` will configure the VRF and source-interface for one of the two options above depending on the value of `default_mgmt_method`.
```

There is no source-interface at play here
```suggestion
                - `use_default_mgmt_method_vrf` will configure the eAPI ACL under VRF for one of the two options above depending on the value of `default_mgmt_method`.
```